notebooks/en/rag_with_sql_reranker.ipynb

```diff
@@ -0,0 +1,849 @@
+{
```

Comment:
<p>For warnings or tips like this (same comment for the other warnings/tips in this notebook), you can use the doc-builder syntax:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&gt; [!WARNING]
&gt; If you are using Google Colab, ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>